nextflow_schema.json

```diff
@@ -432,6 +432,11 @@
                         "greengenes85"
                     ]
                 },
+                "qiime_ref_tax_custom": {
```

Comment:
Usually `--*_ref_tax_custom` comes with a `--*_assign_taxlevels`, is that not needed here?

Reply:
Sorry I didn't reply to this yet. We haven't made use of these flags in our team. If I am understanding the effects of the `--*_assign_taxlevels` flags right, qiime2 doesn't need an equivalent. Qiime2 doesn't take such an argument as Dada does, and it produces a report compatible with downstream analysis by definition so reports wouldn't need to be adapted as the Kraken and Sintax ones are using the same flag.

If I'm mistaken, happy to chat about how to make use of such a flag.